src/sum of digits in number

```diff
@@ -0,0 +1,14 @@
+fn main(){
+    let  x = 234;
+    let a = x%10;
+    println!("a={}",a);
+    let aa = x/10;
+    let s = aa%10;
+    println!("s={}",s);
+    let ss = aa/10;
+    let d = ss%10;
+    println!("d={}",d);
+    let f = a+s+d;
+    println!("the sum is ={}",f);
+    
```

Comment:
1. Please use loops to find sum of digits. The length of number can change and this program is hardcodded for exactly three digits
2. It looks like similar program already exist in repository https://github.com/rustindia/Rust-for-undergrads/blob/master/src/algorithms_data_structures/simple_algorithms/sum_of_digits.rs